controllers/AuthController.js

```diff
@@ -51,36 +49,32 @@ class AuthController {
         const payload = {
           message: 'This email has been taken'
         };
-        return statusResponse.conflict(res, payload);
-      }
-      try {
-        const emailVerification = 'false';
-        const userData = await users.create({
-          email,
-          username,
-          emailVerification,
-          password: hashPassword
-        });
-        await roles.create({
-          role,
-          userId: userData.id
-        });
-        const token = jwt.sign({ email, username }, process.env.TOKEN_SECRET, {
-          expiresIn: 86400
-        });
-        delete userData.dataValues.password;
-        const payload = {
-          message: 'user created succesfully',
-          userData,
-          token,
-          emailToken
-        };
-        return statusResponse.created(res, payload);
-      } catch (error) {
-        return statusResponse.internalServerError(res);
+        return StatusResponse.conflict(res, payload);
       }
+      const emailVerification = 'false';
+      const roleData = await roles.create({
```

Comment:
Hi sunny, I like the speed at which you created this PR, it shows you're working hard. However, I think you can simplify this role and user creation. Since role is the source of user, and user belongs to roles, I think you should be able to do something like this ```roles.create({name: roleName}, {include: [{model: users, as: 'users'}]})```. Kindly research this sequelize method for creating data between connected table. Cheers